Driverless AI/natural-language-processing-tutorial-sentiment-analysis /natural-language-processing-tutorial-sentiment-analysis.md

```diff
@@ -388,11 +426,14 @@ Once the features have been extracted, they can then be used for training a clas
 
 ## Task 4: Driverless AI NLP Recipe
 
-Text data can contain critical information to inform better predictions. H2O Driverless AI automatically converts text strings into features using powerful techniques like TFIDF, CNN, and GRU. With TensorFlow, Driverless AI can also process larger text blocks and build models using all available data to solve business problems like sentiment analysis, document classification and content tagging. In particular, Driverless AI implements the following recipes and models.
+Text data can contain critical information to inform better predictions. H2O Driverless AI automatically converts text strings into features using powerful techniques like TFIDF, CNN, and GRU. Driverless AI version 1.9 introduces support for PyTorch Transformer Architectures (for example, BERT) that can be used for Feature Engineering or as Modeling Algorithms. The Driverless AI platform has the ability to support both standalone text and text with other columns as predictive features. In particular, The following NLP recipes are available for a given text column:
```

Comment:
@Castro38 the last sentence should be rewritten as follows: " In particular, the following NLP recipes are available for a given text column:"

Reply:
@5675sp Done